dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -36,11 +36,17 @@ class LeapHybridSampler(dimod.Sampler):
     Leap’s quantum-classical hybrid BQM solvers are intended to solve arbitrary
     application problems formulated as binary quadratic models (BQM).
 
-    Uses parameters set in a configuration file, as environment variables, or
-    explicitly as input arguments for selecting and communicating with a hybrid
-    solver. For more information, see
+    You can configure your :term:`solver` selection and usage by setting paramters,
+    hierarchically, in a configuration file, as environment variables, or
+    explicitly as input arguments, as described in
     `D-Wave Cloud Client <https://docs.ocean.dwavesys.com/en/stable/docs_cloud/sdk_index.html>`_.
 
+    :ref:`dwave-cloud-client <sdk_index_cloud>`'s
+    :meth:`~dwave.cloud.client.Client.get_solvers` method filters solvers you have
+    access to by `solver properties <https://docs.dwavesys.com/docs/latest/c_solver_3.html>`_
+    ``category=hybrid`` and ``supported_problem_type=bqm``. By default, online
+    hybrid solvers are returned ordered by latest ``version``.
```

Comment:
How about "By default, online hybrid _BQM_ solvers are returned ordered by latest version"? Similarly for DQM below.

Reply:
Sure